src/main/java/mate/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/library_db?serverTimezone=UTC";
```

Comment:
if the init_db.skl file has not changed, then the name db here should be 'taxi; instead 'library_db'